src/App.js

```diff
@@ -24,10 +24,14 @@ function App() {
     <div>
       {/* replace this with <User /> when wanting to test out the user side, we can decide at a later point when the admin side should appear and when the user side should apper */}
       <Switch>
-        <Route path="/" render = {(props) => <User database = {db} />} exact />
-        <Route path="/admin" render = {(props) => <Admin database = {db} />} exact />
+        <Route path="/" render={(props) => <User database={db} />} exact />
+        <Route
+          path="/admin"
+          render={(props) => <Admin database={db} />}
+          exact
+        />
         <Route path="/pricing" component={Pricing} exact />
-        {/* <Route component={Error} /> enable this once there is a 404 page*/}
+        <Route path="/404" component={Error404} exact />
```

Comment:
Change this to path="*"
It needs to show the 404 page on any undefined page not just /404

Reply:
Whoops, my bad. Just made the change :)